wcivf/apps/elections/templates/elections/postcode_view.html

```diff
@@ -99,6 +99,36 @@
 
 
   </div>
+
+  {% if postelection.post.husting_set.count %}
+    <div class="card">
+      <h3>Hustings</h3>
+      {% for husting in postelection.post.husting_set.all %}
+        <h4>{{ husting.title}}</h4>
+      <p>
+        {{ husting.starts | date:"D b d P"}}
```

Comment:
I don't really understand why Django `| date` doesn't implement B as a filter param. Seems obviously useful to me.